src/answers/services/answers.service.ts

```diff
@@ -49,7 +49,7 @@ export class AnswersService {
     user: IUser;
   }): Promise<IAnswer[]> {
     let filter = {};
-    const confirmedUsers = [];
+    const confirmedUsers: (mongoose.Types.ObjectId | string | undefined)[] = [];
     // add current user to users array
     confirmedUsers.push(user.id);
```

Comment:
```suggestion
    confirmedUsers.push(new mongoose.Types.ObjectId(user.id));
```

Reply:
Ah didn't see this, undefined will need to stay though due to line 64